api/src/wfl/module/covid.clj

```diff
@@ -175,6 +175,17 @@
       (update :executor util/to-edn)
       (update :sink     util/to-edn)))
 
+;; Handle `workload` as a raw DB record
+;; (source, executor, and sink not populated)
+;; or a loaded workload object.
+;;
+(defn ^:private workload-to-log [workload]
```

Comment:
Example log for a staged workload:

```
{"message":{"column":5,"namespace":"wfl.server","result":"Updating workload","workload":{"uuid":"8c7dbd7a-1292-422a-95dc-b95f58fb0cb4","executor":"TerraExecutor","labels":["hornet:test","project:(Test) okotsopo/GH-1617-workload-source-logs"],"source":"TerraDataRepoSource","sink":"TerraDataRepoSink"}},"severity":"INFO","logging.googleapis.com/sourceLocation":{"file":"wfl/server.clj","function":"\"Updating workload\"","line":98},"time":"2022-02-17T18:41:32.996305Z"}
```

Reply:
I appreciate the example, and the extra `;;`. Really!
Let's talk about this tomorrow.
Notice the `"function"` label and `sourceLocation`.
We don't need the message string any more.

Reply:
I don't quite follow, but am looking forward to talking!